project/cmake/scripts/common/addon-helpers.cmake

```diff
@@ -34,6 +39,17 @@ macro (build_addon target prefix libs)
     SET_TARGET_PROPERTIES(${target} PROPERTIES PREFIX "lib")
   ENDIF(OS STREQUAL "android")
 
+  GET_TARGET_PROPERTY(LIBRARY_LOCATION ${target} LOCATION)
```

Comment:
I know and I tried `$<TARGET_FILE:${target}>` but it always got interpreted literally with no replacement of the generator.